include/permonmat.h

```diff
@@ -151,6 +151,7 @@ FLLOP_EXTERN PetscErrorCode MatRedistributeRows(Mat mat_from,IS rowperm,PetscInt
 
 /* FETI UTILITIES */
 FLLOP_EXTERN PetscErrorCode MatRemoveGluingOfDirichletDofs(Mat Bgt, Vec cg, Mat Bdt, Mat *Bgt_new, Vec *cg_new, IS *is_new);
+FLLOP_EXTERN PetscErrorCode MatRemoveGluingOfDirichletDofs_old(Mat Bgt, Vec cg, Mat Bdt, Mat *Bgt_new, Vec *cg_new, IS *is_new);
```

Comment:
Could you move it to `include/permon/private/permonmatimpl.h` not to pollute public headers?

Reply:
I think we could merge them or remove the old one entirely. I will put up an issue.

Reply:
I don't remember the history here but I would just make some simple performance comparison to make sure the newer version is at least at the same speed, and then it could be deleted completely.